src/test/java/POM/LoginPage.java

```diff
@@ -0,0 +1,54 @@
+package POM;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class LoginPage extends  basePage{
+    WebDriver driver;
+    WebDriverWait wait;
+
+    //By emailFieldLocator = By.xpath("//input[@type='email']");
+    @FindBy(xpath="//input[@type='email']")
+    WebElement emailField;
+   // By passwordFieldLocator = By.xpath("//input[@type='password']");
+    @FindBy(xpath="//input[@type='password']")
+    WebElement passwordField;
+    By submitButtonLocator = By.xpath("//button[@type='submit']");
+
+
+    public LoginPage(WebDriver givenDriver) {
+        //passing the driver from our loginTests
+        super(givenDriver);
+    }
+
```

Comment:
![image](https://user-images.githubusercontent.com/104761482/200170034-a3ec0821-e11a-47e8-9748-36bf8da1fa67.png)

Here is the fix:

 ```
public LoginPage(WebDriver driver) {
    this.driver = driver;
  }
```